commit 064184d907103ed644c338bce6c3d153d2e631be
Author: Sukadev Bhattiprolu <sukadev@linux.vnet.ibm.com>
Date:   Wed Aug 17 22:41:49 2016 -0400

    UBUNTU: SAUCE: powerpc/pseries: Increase RMA size to 512MB.
    
    BugLink: http://bugs.launchpad.net/bugs/1614309
    
    When trying to boot large kernel and initrd images on large systems
    (with hundreds of CPUs and Terabytes of memory), we sometimes run out
    of memory for the flattened device tree (FDT).
    
    Increase the memory allocated for the Real Mode Area (RMA) to 512MB to
    allow more room for the FDT.
    
    Signed-off-by: Sukadev Bhattiprolu <sukadev@linux.vnet.ibm.com>
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>

diff --git a/arch/powerpc/kernel/prom_init.c b/arch/powerpc/kernel/prom_init.c
index d3eff99..75c3fa7 100644
--- a/arch/powerpc/kernel/prom_init.c
+++ b/arch/powerpc/kernel/prom_init.c
@@ -679,7 +679,7 @@ unsigned char ibm_architecture_vec[] = {
 	W(0xffffffff),			/* virt_base */
 	W(0xffffffff),			/* virt_size */
 	W(0xffffffff),			/* load_base */
-	W(256),				/* 256MB min RMA */
+	W(512),				/* 512MB min RMA */
 	W(0xffffffff),			/* full client load */
 	0,				/* min RMA percentage of total RAM */
 	48,				/* max log_2(hash table size) */
